src/ngraph/frontend/onnx_import/op/add.hpp

```diff
@@ -16,10 +16,11 @@
 
 #pragma once
 
-#include "ngraph/frontend/onnx_import/node.hpp"
 #include "ngraph/node_vector.hpp"
 #include "ngraph/op/add.hpp"
 
+#include "ngraph/frontend/onnx_import/wrappers/node.hpp"
```

Comment:
add `target_include_directory(onnx_import PRIVATE ${CMAKE_CURRENT_SOURCE_DIR})` to `onnx_import/CMakeLists.txt`  and change this to `#include "wrappers/node.hpp"`